pylint_django/compat.py

```diff
@@ -24,5 +24,6 @@
 
 import pylint
 
-# pylint before version 2.3 does not support load_configuration() hook.
-LOAD_CONFIGURATION_SUPPORTED = pylint.__pkginfo__.numversion >= (2, 3)
+# pylint before version 2.3 does not support load_configuration() hook.'
+# The __pkginfo__.numversion attribute was removed in pylint >= 2.8.0
+LOAD_CONFIGURATION_SUPPORTED = pylint.__pkginfo__.numversion >= (2, 3) if hasattr(pylint.__pkginfo__, "numversion") else True
```

Comment:
Instead of defaulting to `True` it could still perform a version check with pylint.__pkginfo__.version 

From pylint [2.8.0 ](https://github.com/PyCQA/pylint/releases/tag/pylint-2.8.0) chages:
`__pkginfo__ now only contain __version__ (also accessible with pylint.__version__), other meta-information are still
accessible with import importlib;metadata.metadata('pylint').`